wallet/crate/src/lib.rs

```diff
@@ -508,14 +453,23 @@ impl Wallet {
 
     /// Returns the current balance associated with this `id`.
     #[inline]
-    pub fn balance(&self, id: AssetId) -> String {
-        self.0.borrow().balance(id.into()).to_string()
+    pub fn balance(&self, id: u32) -> String {
+        // TODO: make id parameter type as u128?
```

Comment:
current still can't use u128 as parameter, it will have compile error even though upgrade wasm-bindgen to `0.2.83`: 
```
the trait `FromWasmAbi` is not implemented for `u128`
```

We can change paramter type to `String`, that'll be ok.

Reply:
String or just do what we do between sdk and runtime and use arrays

Reply:
Can't use `[u8; 32]` as parameter type for `id` for now. It'll also has compile error:
```
 the trait `FromWasmAbi` is not implemented for `[u8; 32]`
```

Yeah, we can change to `String` here, and on manta.js side, passing `string`.

Reply:
change asset_id type to BN on sdk side, but `id` here should be `String`.